src/ui_basic/dropdown.h

```diff
@@ -417,12 +380,23 @@ template <typename Entry> class Dropdown : public BaseDropdown {
 		const Image* img;
 		const std::string tooltip;
 		const std::string hotkey;
+		const HotkeyFunction onShortcutMatch;
```

Comment:
Rename for consistency
```suggestion
		const HotkeyFunction on_shortcut_match;
```